drivers/hmis/app/models/hmis/hud/organization.rb

```diff
@@ -22,6 +22,8 @@ class Hmis::Hud::Organization < Hmis::Hud::Base
   replace_scope :viewable_by, ->(user) do
     ids = user.viewable_organizations.pluck(:id)
     ids += user.viewable_data_sources.joins(:organizations).pluck(o_t[:id])
+    # If a user can see a project within the organization, they can see the organization
+    ids += user.viewable_projects.joins(:organization).pluck(o_t[:id])
```

Comment:
For example, if I can see only one project in the entire HMIS, this is what I see. I do not get to see all the projects in the org, but I get to see the org and it's details.

![Screen Shot 2023-05-01 at 4 59 31 PM](https://user-images.githubusercontent.com/5333982/235535096-1d35f9a5-bb6f-42cf-87f1-ab99ec33f100.png)